### ISSUE TITLE
[Bug]: 'Sofar' - 'HYD15KTL' Connection lost since v2024.02.6

### ISSUE THREAD

**Issue by sahomm** [wills106/homeassistant-solax-modbus#793]
### Describe the bug

First of all, thank you very much for the great work done here!

Since the update to 2024.03.1, Solax no longer receives data via Modbus after a certain period of time without active intervention, e.g. HA homepage closed. Everything is ok for the first few minutes after start/restart Solax, then it stops. I was also able to make this observation with version 2024.06.2. With version 2024.05.X the connection remains stable (see screenshot).
No evaluable data can be found in the log using the search term “solax” or “modbus”.
There are entries here, but they have to do with writing registers, which is a different problem and I would describe it in a new issue after checking.

I suspect that this is related to this #762 

![Bildschirmfoto 2024-03-06 um 09 37 49](https://github.com/wills106/homeassistant-solax-modbus/assets/11404919/2a2c5b8b-2e91-4efb-9fd8-998732a2e8fe)

### Integration Version

2024.03.1

### Homeassistant core version

2024.2.5

### Inverter brand

Sofar

### Plugin used

plugin_sofar.py

### Serial prefix

SP1ES1

### Connection Method

LSE3

**Comment by rcourtman** [comment]
I've noticed a similar issue and it's been happening for me for about two weeks. Unfortunately, I can't be useful and provide version numbers, but I've noticed that all my data just flatlines and I can't find a rhyme or reason as to why. 

![image](https://github.com/wills106/homeassistant-solax-modbus/assets/8825017/3e886207-dd01-4876-a300-66c8563d5450)

Only when I reboot Home Assistant does it pick up again.
